std_msgs/Header header
float64 sensor1
float64 sensor2
